dotCMS/src/main/java/com/dotmarketing/db/DotRunnableThread.java

```diff
@@ -63,4 +81,12 @@ private void internalRunner() {
             }.run();
         }
     }
+    
+    private List<DotRunnable> getFlushers(final List<DotRunnable> allListeners){
+      return allListeners.stream().filter(listener ->listener instanceof FlushCacheRunnable).collect(Collectors.toList());
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Specify a type for: 'listener' [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS2211)